components/common.css

```diff
@@ -1,7 +1,13 @@
+:root {
+  --code-snippet-color: #a7f3d0;
+}
+
 .code-snippet {
   margin: 1rem 0rem;
+  padding: 1rem 2rem;
+  background-color: var(--code-snippet-color);
```

Comment:
use meaningful name for color e.g. color-green-300 (300 as this is lighter shade of green)

Reply:
how to find 300 is the shade number? is there a specific shade number for every color?